public/video-ui/styles/components/_forms.scss

```diff
@@ -219,6 +247,91 @@
   }
 }
 
+.form__field__tags {
+  border-top: 12px solid $color650Grey;
+}
+
+.form__field__tags {
+  border: 1px solid $color400Grey;
+  display: block;
+  border-top: 1px solid $color400Grey;
+  width: 100%;
+  padding: 5px 5px;
+  cursor: pointer;
+}
+
+.form__field__tag__display {
+  text-decoration: underline $textColor;
+}
+
+.form__field--multiselect {
+  $multiSelectBtnWidth: 45px;
+  position: relative;
+  padding-right: $multiSelectBtnWidth + 10px;
+
+  &__btn {
+    width: $multiSelectBtnWidth;
+    position: absolute;
+    right: 0;
+    top: 0;
+    height: 100%;
+    background: $color200Grey;
+    border-left: 1px solid $color300Grey;
+    cursor: pointer;
+  }
+
+  &__options {
+    background: $color200Grey;
+    border: 1px solid $color300Grey;
+    box-shadow: 2px 2px 2px 0px rgba(0,0,0,0.1);
+    position: absolute;
+    top: 48px;
+    left: 0;
+    width: 100%;
+    max-height: 300px;
+    overflow: auto;
+    z-index: 1;
+  }
+
+  &__option {
+    padding: 5px;
+    cursor: pointer;
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $color300Grey;
+    };
+
+    &:hover {
+      background: $color650Grey;
+    }
+  }
+
+  &__value {
+    display: inline-block;
+    padding: 2px 4px;
+    background: $color650Grey;
+    border: 1px solid $color300Grey;
+    border-radius: 2px;
+    margin-right: 2px;
+    font-weight: bold;
+    cursor: pointer;
+    flex-shrink: 0;
+
+
+    &:after {
+      content: '×';
```

Comment:
This is how atom workshop does multiselect elements, I think it is good to have consistent look on the pickers if possible.